integration-test-allow-all-authorizer/src/test/resources/cassandra.yaml

```diff
@@ -122,6 +122,23 @@ authorizer: com.ericsson.bss.cassandra.ecaudit.auth.AuditAuthorizer
 #   increase system_auth keyspace replication factor if you use this role manager.
 role_manager: com.ericsson.bss.cassandra.ecaudit.auth.AuditRoleManager
 
+# Network authorization backend, implementing INetworkAuthorizer; used to restrict user
```

Comment:
Perhaps you already discovered this, but there is a helpful script in bin/import_cassandra_yaml_to_it.sh which can help you to to update config files for integration tests. Just thought I'd mention it.

Reply:
And perhaps the script need some updates to get things right?

Reply:
No, I was not aware of that script, I will look into that, thanks.